projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -56,6 +56,28 @@ function register_extension() {
 	);
 }
 
+/**
+ * Checks if the given URL is valid to
+ * be used in the Spotigy badge.
+ *
+ * @param {string} $url - URL to check.
+ * @return {boolean} True is the URL is valid. Otherwise, False.
+ */
+function is_spotify_url_valid( $url ) {
```

Comment:
I wonder if it would be possible to rely on `Jetpack_Gutenberg::validate_block_embed_url()` instead of something new?

Reply:
I guess the Spotify URL is always a valid RSS resource... but I wonder if there are some conditions where the resource is not.  We really want to lead the users to the URL, regardless of whether is valid or not.

Reply:
addressed here https://github.com/Automattic/jetpack/pull/18456/commits/5ff733e5f38588b6501b71f1e8f2932a55351432